.github/workflows/releaser.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       actions: read # To read the workflow path.
       id-token: write # To sign the provenance.
       contents: write # To add assets to a release.
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.10.0
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@c747fe7769adf3656dc7d588b161cb614d7abfee # v1.10.0
```

Comment:
Does this finally work? There was a long standing bug where pinning by hash resulted in an error downloading the generator script.

Reply:
No idea, let's close this one and I can check it in another repo

Reply:
We can revert if it doesn't work and add it to the exclude list.